src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck: do work");

```

Comment:
The message printed by the `doWork` method should be more informative. It should indicate that a 'Truck' is working, according to the checklist.